### DISCUSSION TITLE
All challenges of a mission unlock when mission completed

### DISCUSSION THREAD

**Discussion by Skylash777** [thepeacockproject/Peacock#518] [Q&A] [answered]
Welcome to my ted talk

**What**
A feature in the options that allows whenever you complete a level you unlock all the challenges. The challenges should be unlocked immediately after you exit the level (after completing the objectives ofc) so you can still do some challenges while in a level and they will be shown as unlocked, but when you finish you also get all the other objectives. This could be just refreshing the game telling it that you did an action, as already unlocked challenges don't recomplete.

**Why**
So I am a perfectionist, and using peacock I try to get all challenges in a level done due to it being so satisfying and nice when I see all the challenges roll in at the end of a mission. However, this usually takes me a LONG while, due to being a casual gamer. The issue is I do not know how to move my progress over to a different peacock version, causing me to have to repeat the same 2 missions over and over. Now, I could just ask or even research a little bit on how to transfer progress, but I have a feeling that my idea will be a lot more fun and less tedious than completing levels over months at a time. If this is too challenging (as I do not know how to code, im bewildered on how yall could even complete this) you can just simply make it easier for people to transfer their progress over peacock versions - or even just tell me how - and that would be fine as well! I do think this will be a good thing for the community though, and you could even specifically change what challenges you get. Thanks, and also awesome mod!

Thank you for listening to my ted talk

**Comment by thecatontheceiling** [comment, marked as answer]
uhhhhhhhhhhh

https://thepeacockproject.org/wiki/intel/updating

**Reply by Skylash777** [reply]
yippee :D
